template/src/extension.ts

```diff
@@ -9,10 +9,14 @@ export function activate(ctx: sourcegraph.ExtensionContext = DUMMY_CTX): void {
     // This is set to an individual language ID by the generator script.
     const languageID = 'all'
 
+    // LSIF is not language-specific, and we only want to initialize it once.
+    // Otherwise we will make a flurry of calls to the frontend to check if
+    // LSIF is enabled.
+    const lsif = initLSIF()
```

Comment:
:+1: and FYI this template extension which bundles all languages only runs during development.